components/das_disconnect_message.py

```diff
@@ -1,29 +1,32 @@
 from components.message import Message
 from canvas import Canvas, Colour
+from data import Data
 
 
 class DASDisconnectMessage(Message):
-    """Class displays error message on overlay when DAS is not
-    connected, or overlay is not connected to an MQTT broker.
-
-    Extends the Message class.
+    """Class displays error message on overlay when DAS is not connected, or
+    overlay is not connected to an MQTT broker.
 
     Attributes:
         text_colour: Colour type representing the colour of the message
+        message: String representing the error message that displays on the
+        overlay
+        client: MQTT Client which is used to check if client is connected
     """
 
+    # These class variables must be consistent for every instance
```

Comment:
To be pedantic, I don't no if this comment is necessary - this is exactly what a class variable is

Reply:
Ah I wrote that to explain why `text_colour` and `message` is a class variable rather than being instantiated as an instance variable.

Reply:
Hm i suppose this is obvious anyways, all good.